requirements.txt

```
@@ -1,3 +1,3 @@
 pip~=23.3
 poetry~=1.7
-setuptools~=69.0
+setuptools>=70.0.0
```

Is being more explicit better?
```suggestion
setuptools~=75.2.0
```